### ISSUE TITLE
.tse directory not create automatically

### ISSUE THREAD

**Issue by Monirzadeh** [ghodsizadeh/tehran-stocks#82]
if `.tse` not exist program can't create that on Arch linux

**Comment by ghodsizadeh** [comment]
Can you show me the error when there is no `.tse` file.

**Comment by Monirzadeh** [comment]
> Can you show me the error when there is no `.tse` file.

Traceback (most recent call last):
  File "/home/MYUSERNAME/PROJECT/init.py", line 2, in <module>
    import tehran_stocks
  File "/home/MYUSERNAME/PROJECT/lib/python3.10/site-packages/tehran_stocks/__init__.py",
line 3, in <module>
    import tehran_stocks.config as db
  File "/home/MYUSERNAME/PATHTOVIRTUALENV/.virtualenvs/stock/lib/python3.10/site-packages/tehran_stocks/config/__init__.py", line 1, in <module>
    from .engine import *
  File "/home/MYUSERNAME/PATHTOVIRTUALENV/.virtualenvs/stock/lib/python3.10/site-packages/tehran_stocks/config/engine.py", line 21, in <module>
    with open(CONFIG_PATH, "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/MYUSERNAME/.tse/config.yml'

**Comment by sfmqrb** [comment]
I have the same problem on the GNU/Linux Ubuntu 21.10.
Also I reinstall package completely however my problem persists!
@ghodsizadeh

**Comment by ghodsizadeh** [comment]
Could you please check with version 1.0.5 and let me know if the error still exists?

**Comment by Monirzadeh** [comment]
> Could you please check with version 1.0.5 and let me know if the error still exists?

yes it is still exist with this error
```
Traceback (most recent call last):
  File "/home/pathtoscript/init.py", line 2, in <module>
    from tehran_stocks import Stocks
  File "/home/pathtovir/lib/python3.10/site-packages/tehran_stocks/__init__.py",
line 3, in <module>
    import tehran_stocks.config as db
  File "/home/pathtovir/lib/python3.10/site-packages/tehran_stocks/config/__init__.py", line 1, in <module>
    from .engine import *
  File "/home/pathtovir/lib/python3.10/site-packages/tehran_stocks/config/engine.py", line 25, in <module>
    create_config()
  File "/home/pathtovir/lib/python3.10/site-packages/tehran_stocks/config/engine.py", line 22, in create_config
    with open(CONFIG_PATH, "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/myusername/.tse/config.yml'
```

**Comment by najafi82** [comment]
run again with administrative privilege, it creates directory automatically.

**Comment by amirbadzohreh** [comment]
Same Problem!!
[Errno 2] No such file or directory: '/Users/amir/.tse/config.yml'

**Comment by peima** [comment]
Same Problem!!
[Errno 2] No such file or directory: '/Users/.tse/config.yml'

**Comment by ghodsizadeh** [comment]
It seems the bug is fixed in #85.
Please upgrade the package and test it in  your operating systems:
```bash
$ pip install -U tehran_stocks
$ python -c "from tehran_stocks import db"
```
